stock_request/models/stock_request.py

```diff
@@ -149,6 +170,62 @@ def _check_qty(self):
                     _("Stock Request product quantity cannot be negative.")
                 )
 
+    @api.depends("order_id.warehouse_id")
+    def _compute_warehouse_id(self):
+        for request in self:
+            if request.order_id.warehouse_id:
+                request.warehouse_id = request.order_id.warehouse_id
+            else:
+                request.warehouse_id = request.warehouse_id
+
+    @api.depends("order_id.location_id")
+    def _compute_location_id(self):
+        for request in self:
+            if request.order_id.location_id:
+                request.location_id = request.order_id.location_id
+            else:
+                request.location_id = request.location_id
+
+    @api.depends("order_id.requested_by")
+    def _compute_requested_by(self):
+        for request in self:
+            if request.order_id.requested_by:
+                request.requested_by = request.order_id.requested_by
+            else:
+                request.requested_by = request.requested_by
+
+    @api.depends("order_id.expected_date")
+    def _compute_expected_date(self):
+        for request in self:
+            if request.order_id.expected_date:
+                request.expected_date = request.order_id.expected_date
+            else:
+                request.expected_date = request.expected_date
+
+    @api.depends("order_id.picking_policy")
+    def _compute_picking_policy(self):
+        for request in self:
+            if request.order_id.picking_policy:
+                request.picking_policy = request.order_id.picking_policy
+            else:
+                request.picking_policy = request.picking_policy
+
+    @api.depends("order_id.procurement_group_id")
+    def _compute_procurement_group_id(self):
+        for request in self:
+            if request.order_id.procurement_group_id:
+                request.procurement_group_id = request.order_id.procurement_group_id
+            else:
+                request.procurement_group_id = request.procurement_group_id
+
+    @api.depends("order_id.company_id")
+    def _compute_company_id(self):
+        for request in self:
```

Comment:
@petrus-v Could `request` and `request order` be in different companies????

Reply:
I don't think so because of the existing constraint: https://github.com/OCA/stock-logistics-warehouse/pull/2309/files#diff-7da7c1c93876e1d2eeebd0827ded3675f1d2603621937dfc1ea551847d8672b2L259

a `stock.request` can leave alone without `stock.request.order`.


I'm not sure what's the point?